app/src/main/java/com/iic/mokojin/CurrentMatchFragment.java

```diff
@@ -0,0 +1,186 @@
+import android.util.Log;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.iic.mokojin.models.Match;
+import com.iic.mokojin.models.Models;
+import com.iic.mokojin.models.Player;
+import com.iic.mokojin.operation.EndMatchOperation;
+import com.parse.ParseException;
+
+import bolts.Continuation;
+import bolts.Task;
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import butterknife.OnLongClick;
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class CurrentMatchFragment extends Fragment {
+
+    private static final String LOG_TAG = CurrentMatchFragment.class.getName();
+
+    private Match mCurrentMatch;
+
+    @InjectView(R.id.empty_text_view)  View mEmptyText;
+    @InjectView(R.id.players_container) View mPlayersContainer;
+
+    @InjectView(R.id.player_a_name)  TextView mPlayerANameTextView;
+    @InjectView(R.id.player_a_image_front) ImageView mPlayerAImageViewFront;
+    @InjectView(R.id.player_a_image_back) ImageView mPlayerAImageViewBack;
+
+    @InjectView(R.id.player_b_name)  TextView mPlayerBNameTextView;
+    @InjectView(R.id.player_b_image_front) ImageView mPlayerBImageViewFront;
+    @InjectView(R.id.player_b_image_back) ImageView mPlayerBImageViewBack;
+
+    public CurrentMatchFragment() {
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.fragment_current_match, container, false);
+        ButterKnife.inject(this, rootView);
+
+        return rootView;
+    }
+
+    private void refreshCurrentMatch() {
+        mCurrentMatch = null;
+        Match.getCurrent().continueWith(new Continuation<Match, Void>() {
+            @Override
+            public Void then(Task<Match> task) throws Exception {
+                if (task.isCancelled()) {
+                    Log.d(LOG_TAG, "Fetching of current match was cancelled");
+                } else if (task.isFaulted()) {
+                    Log.e(LOG_TAG, "Error fetching current match", task.getError());
+                } else {
+                    mCurrentMatch = task.getResult();
+                    refreshUI();
+                }
+                return null;
+            }
+        }, Task.UI_THREAD_EXECUTOR);
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        refreshCurrentMatch();
+    }
+
+    private void displayCharacter(int characterId, ImageView imageView) {
+        int resourceId = getResources().getIdentifier("player_" + characterId, "drawable", this.getActivity().getPackageName());
+        Drawable charDrawable = getResources().getDrawable(resourceId);
+        imageView.setImageDrawable(charDrawable);
```

Comment:
you can inline this method + us the utiltty for it + use setImageResource instead of loading the drawable